test/gui/shared/scripts/helpers/SyncHelper.py

```diff
@@ -1,17 +1,39 @@
 import re
 import sys
 import test
-from os.path import realpath
+import os
+import urllib
 from squish import waitFor, snooze
 
+from helpers.FilesHelper import sanitizePath
+from helpers.ConfigHelper import get_config
+
+# NOTE: 'syncstate.py' was removed from client
+# and is now available at 'owncloud/client-desktop-shell-integration-nautilus'
+# check if 'syncstate.py' is available, if not, download it
+custom_lib = get_config('custom_lib')
+syncstate_lib_file = os.path.join(custom_lib, 'syncstate.py')
+if not os.path.exists(custom_lib):
+    os.makedirs(custom_lib, exist_ok=True)
+if not os.path.exists(syncstate_lib_file):
+    URL = "https://raw.github.com/owncloud/client-desktop-shell-integration-nautilus/master/src/syncstate.py"
```

Comment:
Please use a lower-case variable name like `url`. Using uppercase only for variable names violates Python conventions.